x11-plugins/purple-slack/purple-slack-9999.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3
+
+DESCRIPTION="A libpurple plugin for the web-based slack api."
+HOMEPAGE="https://github.com/dylex/slack-libpurple"
+EGIT_REPO_URI="https://github.com/dylex/slack-libpurple"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE=""
```

Comment:
I find it kinda inconsistent that you keep empty `IUSE` but remove empty `KEYWORDS`.